server/serializers/twitter-user.go

```diff
@@ -0,0 +1,8 @@
+package serializers
```

Comment:
Consider this analogous to the `mattermost-server/model` package.

It has structs that will be used for storing data in the DB, but also used elsewhere in the app.
The reason I am not using the name `model` is to avoid name conflict when also using the `mattermost-server/model`.